SwiftyBluetooth/Source/Central.swift

```diff
@@ -144,6 +156,20 @@ extension Central {
         return self.centralProxy.centralManager.isScanning
     }
     
+    /// Alternative connection method, when identifier and services are known for a peripheral.
+    ///
+    /// - Parameter peripheralUUID: The unique identifier (UUID) for the given peripheral.
+    /// - Parameter serviceUUIDs: The service UUIDs to for the given peripheral.
+    /// - Parameter timeout: The time in seconds before the connection attempt is stopped and the completion closure is called with a failed result.
+    /// - Parameter completion: The closures, called upon completion (succesful or otherwise) of the connection.
+    public func connect(peripheralUUID: UUID,
```

Comment:
You're going way out of bounds with your comparaison, we're not in the same ballpark whatsoever. 

The difference between looking if a peripheral is already in the connected peripheral list before connecting and adding an API that let you do things which you should never ever be doing is kind of major. 

The fact of the matter is saving the UUID of peripherals and trying to reconnect to them based on that is just completely wrong. You should be using the CBCentralManagerState restore state functions if you need these peripherals between runs and, when you can't do that, re-identify peripherals using their advertisement data and by reading standard bluetooth characs/descriptors on them or by implementing your own protocol between your iOS device and the Peripherals you're interested in. There is no guarantee whatsoever that the CBUUID of peripherals will not change between runs and, while it may be convenient for you, and work for you, to do it that way does not mean it's how it's supposed to be done.